suite-common/icons/generateIcons.js

```diff
@@ -96,6 +91,23 @@ const generateIconsFileContent = assetTypesArray => {
     // If you want to add of modify icons please read README.md to find out how to do it
 
     ${mappedAssetTypes.join('')}
+
+    export const tokenIcons = {
```

Comment:
It has to be hardcoded, because the key (contract) does not match icon filename anymore.

This change is not related to this PR, but I had to fix this because, after calling `yarn generate-icons` the token icons disappered.

Reply:
If this is defined by hand I would prefer to new file for it that is not generated.

Reply:
moved to separate file in 814aa2782